packages/web-components/src/components/va-link-action/va-link-action.tsx

```diff
@@ -0,0 +1,91 @@
+import { Component, Event, EventEmitter, Host, h, Prop } from '@stencil/core';
+import classNames from 'classnames';
+
+/**
+ * @componentName Link - Action (new)
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+@Component({
+  tag: 'va-link-action',
+  styleUrl: 'va-link-action.css',
+  shadow: true,
+})
+export class VaLinkAction {
+  /**
+   * If `true`, the component-library-analytics event is disabled.
+   */
+  @Prop() disableAnalytics?: boolean = false;
+
+  /**
+   * The href attribute of the anchor.
+   */
+  @Prop() href!: string;
+
+  /**
+   * The anchor text.
+   */
+  @Prop() text!: string;
+
+  /**
+   * The type of the link, which determines its style.
+   * Can be 'primary', 'secondary', or 'reverse'.
+   */
+  @Prop() type: "primary" | "secondary" | "reverse" = "primary";
+
+  /**
+   * The event used to track usage of the component.
+   */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'component-library-analytics',
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  /** @ts-ignore */
+  private handleClick = (e: MouseEvent): void => {
+    if (!this.disableAnalytics) {
+      const detail = {
+        componentName: 'va-link-action',
```

Comment:
Nice! A follow-up task to this would be for us to let the analytics team know that this component is new and will need to be tracked. And also add it to the [component-library-analytics-setup.js](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/site-wide/component-library-analytics-setup.js) file. Maybe those things can be a follow-up ticket?

Reply:
I see that is the part of the ticket, I'll be making that changes today as its simple task.

Reply:
PR for updating the analytics script: https://github.com/department-of-veterans-affairs/vets-website/pull/30196